gaps/gap-6_tech_council_process/gap-6_tech_council_process.md

```diff
@@ -0,0 +1,79 @@
+---
+gap: 6
+title: Tech Council process
+description: Rules of operation for Golem Tech Council
+author: stranger80 (@stranger80)
+status: Draft
+type: Meta
+---
+
+## Abstract
+This is proposal to structure the proceedings of Golem Tech Council in order to streamline the flow of agenda items and make the approval process efficient.
+
+## Motivation
+The purpose is to ensure the Tech Council operates efficiently, with as few synchronous actions (like meetings) as possible, and with transparency to all Golem Factory team members.
+
+## Specification
+
+The main purpose of this proposed process is to make proceedings of Tech Council items more efficient and asynchronous (eg. have them raised and reviewed offline, and only use Tech Council sessions to discuss questions on items which have been already digested by participants). 
+
+This is why responsibilities of the Facilitator also must include upfront facilitation of Tech Council session agenda, so that the participants aregiven an early heads-up and are able to prepare for the session (eg. by reading and reviewing items proposed on the agenda).
+
+This section specifies the rules for Tech Council proceedings, by defining key elements and workflow.
+
+### Tech Council Members
+
+Two roles of Tech Council members are specified:
+- **Participant** - participant of Tech Council proceedings and sessions
```

Comment:
I'd highlight that any1 from within the Golem Factory can be a **participant**. I would say we want to open #tech-council channel @ our internal instant messenger. Notion page is already shared with all GF members.

Reply:
OK, adjusted.